getting-started-tutorial/tutorial-1-MLRun-basics.ipynb

```diff
@@ -0,0 +1,1890 @@
+{
```

Comment:
<p>typo: We are saving to data container "Projects" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/demos/pull/73/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>